<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1071</width>
    <height>394</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>JoyStream</string>
  </property>
  <property name="iconSize">
   <size>
    <width>32</width>
    <height>32</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="addTorrentFilePushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>15</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add torrent file</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="earnedBalanceLabel">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="earnedTitleQlabel">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>44</y>
      <width>41</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>earned</string>
    </property>
   </widget>
   <widget class="QLabel" name="spentTitleLabel">
    <property name="geometry">
     <rect>
      <x>795</x>
      <y>44</y>
      <width>35</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>spent</string>
    </property>
   </widget>
   <widget class="QLabel" name="spentBalanceLabel">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceLabel">
    <property name="geometry">
     <rect>
      <x>954</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="balanceTitleLabel">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>44</y>
      <width>45</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>balance</string>
    </property>
   </widget>
   <widget class="QPushButton" name="walletPushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>400</y>
      <width>151</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>wallet</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QTreeView" name="torrentsTable">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>80</y>
      <width>1075</width>
      <height>239</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="indentation">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="updatePushButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>update</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>791</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>This version only uses Bitcoin testnet. Keep running at all times for testing purposes, restart for automatic updates.</string>
    </property>
   </widget>
   <widget class="QLabel" name="unconfirmedBalanceLabel">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>16</y>
      <width>100</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>N/A</string>
    </property>
   </widget>
   <widget class="QLabel" name="unconfirmedBalanceTitleLabel">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>44</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>unconfirmed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bugsPushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>330</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>report bugs</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addMagnetLinkPushButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>15</y>
      <width>161</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>add magnet link</string>
    </property>
    <property name="iconSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="topUpWalletPushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>400</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>add funds</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
